.gitignore

```diff
@@ -33,6 +33,9 @@ docs/_build
 coverage.xml
 htmlcov
 
+# our nix setup produces this
+process-compose.socket
```

Comment:
Out of curiosity, what was your setup, Windows 11 with WSL 2? I think I have never seen this file mentioned by git. On my Linux, it's a socket, and git auto-ignores it.